### PR TITLE
Merge Branch-RefactorAddCommand-Hong Liang with Master

### LINKED ISSUE #76: Rename add command to addPerson
Change command word of add person command from `add` to `addPerson`.

### PR DESCRIPTION
**Pull request by tanhl2000**
closes #76

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddPersonCommand.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /**
  * Adds a person to the address book.
  */
-public class AddCommand extends Command {
+public class AddPersonCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
```
Comment (Qiaoran-M) [inline src/main/java/seedu/address/logic/commands/AddPersonCommand.java]: You can change the command word here
```suggestion
    public static final String COMMAND_WORD = "addPerson";
```

File: src/test/java/seedu/address/testutil/PersonUtil.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public class PersonUtil {
      * Returns an add command string for adding the {@code person}.
      */
     public static String getAddCommand(Person person) {
```
Comment (Qiaoran-M) [inline src/test/java/seedu/address/testutil/PersonUtil.java]: you can change the method signature to `getAddPersonCommand` as well.
```suggestion
    public static String getAddPersonCommand(Person person) {
```

**Review by Qiaoran-M — CHANGES_REQUESTED** [review]
LGTM except minor issues!

File: src/main/java/seedu/address/logic/commands/AddPersonCommand.java (hunk 1 above)
Comment (tanhl2000) [reply src/main/java/seedu/address/logic/commands/AddPersonCommand.java]: ok will do!

File: src/test/java/seedu/address/testutil/PersonUtil.java (hunk 2 above)
Comment (tanhl2000) [reply src/test/java/seedu/address/testutil/PersonUtil.java]: thanks for catching!

**Review by Qiaoran-M — APPROVED** [review]
LGTM!

**Pull request merged**